### PR TITLE
fix: [PIE-5433]: Azure artifact icons

### PR DESCRIPTION
**Pull request by satyam-harness**
You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

File: packages/icons/src/service-azure-artifact-connector.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" data-name="fluent_icons" viewBox="0 0 18 18"><defs><linearGradient id="a" x1="9" y1="16.97" x2="9" y2="1.03" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#0078d4"/><stop offset=".16" stop-color="#1380da"/><stop offset=".53" stop-color="#3c91e5"/><stop offset=".82" stop-color="#559cec"/><stop offset="1" stop-color="#5ea0ef"/></linearGradient></defs><path data-name="Path 1" d="M17 4v9.74l-4 3.28-6.2-2.26V17l-3.51-4.59 10.23.8V4.44zm-3.41.49L7.85 1v2.29L2.58 4.84 1 6.87v4.61l2.26 1V6.57z" fill="url(#a)"/></svg>
```
Comment (ishantmehtaharness) [inline packages/icons/src/service-azure-artifact-connector.svg:1]: Avoid LinearGradient and other filters as they [don’t work in Safari](https://stackoverflow.com/questions/48836885/large-blurs-in-svg-filters-not-working-in-safari).
https://harness.atlassian.net/wiki/spaces/PLATFORM/pages/1963262066/SVG+Guidelines+Best+Practices

File: packages/icons/src/service-azure-artifacts.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...://www.w3.org/2000/svg" fill="none" viewBox="0 0 49 49"><path d="M18.465 48.656H2.402a2.103 2.103 0 01-2.093-2.104V26.75c0-1.16.942-2.094 2.093-2.094h16.063v24z" fill="#B41055"/><path d="M46.255 48.656H39.45v-24h6.806c1.131 0 2.054.923 2.054 2.055V46.6a2.058 2.058 0 01-2.054 2.055z" fill="#E86299"/><path d="M18.613 48.656h21.768v-24H16.39v21.777c0 1.22.992 2.223 2.222 2.223z" fill="#F173A7"/><path d="M38.199.656H10.657c-1.19 0-2.153.963-2.153 2.154v21.846h31.848V2.81A2.151 2.151 0 0038.199.656z" fill="#C12664"/><path d="M32.365.656H10.657c-1.19 0-2.153.963-2.153 2.154v21.846h23.861v-24z" fill="#CF3071"/><path d="M22.522 8.597h-4.167a1.99 1.99 0 01-1.984-1.985 1.99 1.99 0 011.984-1.985h4.167a1.984 1.984 0 110 3.97z" fill="#B5205C"/><path d="M32.313 38.64H20.438v2.035h11.876v-2.034zm-5.119 4.042h-6.756v2.034h6.756v-2.034z" fill="#FE90BE"/><path d="M30.382 32.715h-3.959a2.063 2.063 0 01-2.064-2.065c0-1.141.923-2.064 2.064-2.064h3.959c1.14 0 2.063.923 2.063 2.065a2.063 2.063 0 01-2.063 2.064z" fill="#CB2E6F"/><path d="M14.276 32.715h-3.958a2.063 2.063 0 01-2.064-2.065c0-1.141.923-2.064 2.064-2.064h3.958c1.141 0 2.064.923 2.064 2.065a2.063 2.063 0 01-2.064 2.064z" fill="#900740"/></svg>
```
Comment (ishantmehtaharness) [inline packages/icons/src/service-azure-artifacts.svg:1]: fill="none" should not be present on the SVG element

File: packages/icons/src/service-azure-artifact-connector.svg (hunk 1 above)
Comment (satyam-harness) [reply packages/icons/src/service-azure-artifact-connector.svg:1]: actually i removed whole def tag but svg was not rendering at all after that

File: packages/icons/src/service-azure-artifacts.svg (hunk 2 above)
Comment (satyam-harness) [reply packages/icons/src/service-azure-artifacts.svg:1]: roger

**Review by kapilkumawat86 — DISMISSED** [review]

**Review by ishantmehtaharness — APPROVED** [review]
Approving
please check if we can remove `fill: none`

**Comment by ishantmehtaharness** [comment]
Looks good

**Review by ishantmehtaharness — DISMISSED** [review]

**Review by rameshx — APPROVED** [review]

**Review by depanshu-dhiman — APPROVED** [review]

**Pull request merged**